Automattic/pocket-casts-android
# Sync auto archive podcast settings

## Description

One of the settings listed for sync project #1739. This PR adds syncing to the podcast settings for auto archive configuration.

## Testing Instructions

> [!note]
> You need to test this with staging because changes aren't deployed to production, yet.
> 
> To test on staging use the `debug` variant instead of the `debugProd` one.

I won't provide step by step instructions here as the auto archive controls consist of 4 separate settings. You should check if auto archiving configuration syncs correctly. You can reach them under `Podcasts`/`podcast`/`Settings (cog icon)`/`Auto archive`.

## Checklist
 
#### I have tested any UI changes...
